src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,19 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+        return findIndex(records, email);
+    }
+
+    private int findIndex(String[] records, String email) {
+        for (String record : records) {
+            if (record.contains(email) && !record.replace(email, "").isEmpty()) {
```

Comment:
let's simplify solution instead of performing replace + substring, use single split(":") operation. Result save to String[] devidedRecord